code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -51,12 +51,22 @@
 	make_reagents = list()
 	category = list("initial","Food")
 
-/datum/design/enzyme
-	name = "10u Universal Enzyme"
-	id = "enzyme"
+/datum/design/bbq_bottle
```

Comment:
No, you can already make bbq sauce with what can found between the kitchen and botany already.
Also as I told someone before, the build paths should be reagents, not containers.